lib/ex_queue_bus_client.ex

```diff
@@ -11,10 +11,38 @@ defmodule ExQueueBusClient do
     SQS.send_message(queue(), action)
   end
 
+  @doc """
+  AWS SQS queue name to use if SQS is configured.
+  """
+  @spec queue() :: binary
   def queue do
     Application.get_env(:ex_queue_bus_client, :queue) || "probando"
```

Comment:
What do you think about removing this "probando" default

Reply:
I agree.